### ISSUE TITLE
No new line nor correction dialog announcement

### ISSUE THREAD

**Issue by josephsl** [dictationbridge/dictationbridge-jfw#4]
Hi,

Reported by a user: after dictating, JAWS does not announce new line or correction dialog.

STR:

1. Run JAWS with DB installed.
2. Dictate some text and say, "new line" or "correct that".

Expected: new line and/or correction dialog should be announced.
Actual: no announcement.

Thanks.
